.changes/v1.11/BUG FIXES-20250206-155025.yaml

```diff
@@ -0,0 +1,5 @@
+kind: BUG FIXES
+body: Fixes hanging behavior seen when applying a saved plan with -auto-approve using the cloud backend
```

Comment:
Although there are no special characters, I am somewhat surprised this isn't quoted 🤔 but it appears that changie dry run check is happy so no biggie 🤷🏻

Reply:
YAML 🤷 😂 

Thanks for checking